core/node/state_keeper/src/metrics.rs

```diff
@@ -30,6 +30,40 @@ pub enum TxExecutionType {
     L2,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, EncodeLabelValue)]
+#[metrics(rename_all = "snake_case")]
+pub enum TxExecutionStatus {
+    Success,
+    Rejected,
+    Reverted,
+    Halted,
```

Comment:
If you apply my suggestion from the previous comment, then `Halted` variant won't be needed:
- all txs that result into `Halted` are either rejected or rolled back
- all txs that are added to `UpdatesManager` must not have `Halted` status, I suggest asserting it